### ISSUE TITLE
JavaScript GPUdb.js can't be compiled 

### ISSUE THREAD

**Issue by petrvecera** [kineticadb/kinetica-api-javascript#5]
https://github.com/kineticadb/kinetica-api-javascript/blob/master/javascript/GPUdb.js#L321

```
./src/GPUdb.js
  Line 321:   'url' is not defined  no-undef
  Line 323:  'url' is not defined  no-undef
  Line 328:   'url' is not defined  no-undef
  Line 330:  'url' is not defined  no-undef
```

Indeed this variable is not defined anywhere.

**Comment by mmahmud** [comment]
Hello Petr,

Thank you for bringing to this our attention; will open a bug report for
this and fix it soon.

**Comment by kylesutton** [comment]
Addressed in the latest release (7.1.8.2): https://github.com/kineticadb/kinetica-api-javascript/tree/release/v7.1
